### PR TITLE
WiSE Page

### PR DESCRIPTION
**Pull request by alexgirard**
- added WiSE page
- updated action prompt button component for reuse, still the same on the Get Involved page
- fixed some console errors

**desktop**
wise

### CODE REVIEW

File: src/components/BlogPostsGroup.jsx [hunk 1]
```diff
@@ -35,12 +35,12 @@ const BlogTile = ({color, type, title, image, links}) => {
 
 BlogTile.propTypes = {
   color: PropTypes.string.isRequired,
-  type: PropTypes.oneOf(Object.values(BLOG_TYPES)),
+  type: PropTypes.oneOf([...Object.values(BLOG_TYPES)]),
```
Comment (RobbieZhuang) [inline src/components/BlogPostsGroup.jsx:38]: Just curious: Why do we add `[...` `]`.

File: src/components/styles/PromptActionButtons.scss [hunk 2]
```diff
@@ -1,8 +1,10 @@
 @import "../../styles/colors";
-.action-row {
-    height: 60px;
-    display: flex;
-    align-items: center;
+
```
Comment (RobbieZhuang) [inline src/components/styles/PromptActionButtons.scss:2]: Thank u for fixing my jank css <3. Just make sure things look gucci in mobile/iPhone 6 size on `Get Involved` & `WiSE` pages. Thanks!

File: src/components/styles/PromptActionButtons.scss [hunk 3]
```diff
@@ -13,7 +15,7 @@
     float: right;
     background-color: $color-math;
     border-width: 0px;
-    width: 150px;
+    // max-width: 200px;
```
Comment (RobbieZhuang) [inline src/components/styles/PromptActionButtons.scss:18]: nit: can remove if we're not using this

File: src/containers/Wise.jsx [hunk 4]
```diff
@@ -0,0 +1,316 @@
+import React, {Component, useState} from "react";
+import {Container, Row, Col, Button, Accordion, Card} from "react-bootstrap";
+import url from "url";
+import PropTypes from "prop-types";
+import Footer from "../components/Footer.jsx";
+import PromptActionButtons from "../components/PromptActionButtons.jsx";
+import "./styles/Wise.scss";
+
+const DISCORD_LINK = "https://discord.gg/MRQq7H";
```
Comment (RobbieZhuang) [inline src/containers/Wise.jsx:9]: Discord link says `Invite Invalid`

File: src/containers/Wise.jsx [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+    ),
+  },
+  {
+    question: "What events do you run each term?",
+    answer: (
+      <p>
+        Events vary from term to term, but in the past we have run a brunch,
+        various mentoring activities, several casual board game nights, and a
+        “How to get an awesome co-op” workshop. Check out the events page WiSE
+        calendar for more information about our upcoming events!
```
Comment (RobbieZhuang) [inline src/containers/Wise.jsx:65]: Is there something separate on the events page for WiSE?

File: src/containers/Wise.jsx [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
+          the majority of attendants will be male. We hold women-only events to
+          lessen this issue.
+        </p>
+        <p className="pt-3">
+          Following in the footsteps of{" "}
+          <ExternalLink href="http://wics.uwaterloo.ca/">WiCS</ExternalLink>, we
+          hold women-only technical events to reduce the feeling of{" "}
+          <ExternalLink href="http://geekfeminism.wikia.org/wiki/Letting_the_side_down">
+            letting the side down
+          </ExternalLink>
```
Comment (RobbieZhuang) [inline src/containers/Wise.jsx:86]: @RobbieZhuang make a note to potentially use <ExternalLink> tag everywhere throughout the rest of the website.

File: src/containers/Wise.jsx [hunk 7]
```diff
@@ -0,0 +1,316 @@
...
+};
+
+const Resources = () => (
+  <>
+    <h2 className="pt-4 pb-2">Resources</h2>
+    <p>
+      For more information on events or scholarship opportunities please
+      checkout our{" "}
+      <a
+        href="https://www.notion.so/WiSE-Resource-Hub-096fb9b198ad4cf68891847f25960a01"
```
Comment (RobbieZhuang) [inline src/containers/Wise.jsx:230]: OMG I love the notion!!!

File: src/containers/Wise.jsx [hunk 8]
```diff
@@ -0,0 +1,316 @@
...
+        target="_blank"
+        rel="noopener noreferrer"
+        className="text-nowrap"
+      >
+        resource hub ↗
```
Comment (RobbieZhuang) [inline src/containers/Wise.jsx:235]: Would you be okay with removing `↗` for consistency with other links?

File: src/components/BlogPostsGroup.jsx (hunk 1 above)
Comment (alexgirard) [reply src/components/BlogPostsGroup.jsx]: Not needed! That was late night alex 🙃

File: src/containers/Wise.jsx (hunk 4 above)
Comment (alexgirard) [reply src/containers/Wise.jsx]: Fixed!

File: src/containers/Wise.jsx (hunk 5 above)
Comment (alexgirard) [reply src/containers/Wise.jsx:70]: Updated to add an event calendar on the page, just need to add env vars (assuming you would have to do that? not sure)

File: src/components/ProfilePhotoGroup.jsx [hunk 9]
```diff
@@ -51,7 +51,7 @@ class ProfilePhotoGroup extends Component {
         ) : (
           <h3>{soc} Execs</h3>
         )}
-        <Row className="mb-3">
+        <Row className="mb-3 ml-0">
```
Comment (alexgirard) [inline src/components/ProfilePhotoGroup.jsx:54]: small bug that cuts off left bit of card on about us page

File: src/components/styles/BlogPostsGroup.scss [hunk 10]
```diff
@@ -60,7 +60,8 @@
 }
 
 .card-body {
-  padding: 10px 14px;
+  padding: 10px 14px !important;
```
Comment (alexgirard) [inline src/components/styles/BlogPostsGroup.scss:63]: another small formatting bug

File: src/containers/Wise.jsx (hunk 5 above)
Comment (RobbieZhuang) [reply src/containers/Wise.jsx]: Curious which env vars do you need to add? But feel free to add them in :)

File: src/components/ProfilePhotoGroup.jsx (hunk 9 above)
Comment (RobbieZhuang) [reply src/components/ProfilePhotoGroup.jsx:54]: Thank you!!!

**Comment by RobbieZhuang** [comment]
Thanks for the fixes! Feel free to merge 🚀

File: src/containers/Wise.jsx (hunk 5 above)
Comment (alexgirard) [reply src/containers/Wise.jsx]: For the wise calendar! Can you add them for me pls 😄  I'll slack them to you

**Pull request merged**